.github/workflows/validate_kong_image.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
       - uses: actions/checkout@v3
       - run: |
           gh issue comment ${ISSUE_NUMBER} --body \
-            'Kong Gateway validation tests were started at ${URL} with the following parameters:
+            'Kong Gateway validation tests were started at ${{ env.URL }} with the following parameters:
```

Comment:
Thanks for fixing this 😅 I forgot when changing this in #4031 that `''` wouldn't expand env variables

🎖️